org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/text/correction/ActionMessages.properties

```diff
@@ -12,11 +12,11 @@
 ###############################################################################
 
 OverrideMethodsAction_label=Override/Implement Methods...
-GenerateGetterSetterAction_label=Generate Getters and Setters
+GenerateGetterSetterAction_label=Generate Getter and Setter
```

Comment:
It's better to define a new template message for the new getter and setter quickfix, and use `Messages.format()` to replace the variable.

GenerateGetterSetterAction_templateLabel=Generate Getter and Setter for ''{0}''
GenerateGetterAction_templateLabel=Generate Getter for ''{0}''
GenerateSetterAction_templateLabel=Generate Setter for ''{0}''

Reply:
Great. I just keep the current messages and define templates used by quick assists only via https://github.com/eclipse/eclipse.jdt.ls/pull/2092/commits/46d3425026d658e10cea5861812de7e945e8bb2e